### ISSUE TITLE
`parsePhoneNumberFromString` issue with E164 format for an Australian number

### ISSUE THREAD

**Issue by imranariffin** [catamphetamine/libphonenumber-js#375]
```js
import { parsePhoneNumberFromString } from 'libphonenumber-js/mobile'
const parsed = parsePhoneNumber('61438331999', 'AU')
console.log(parsed.format('E.164'))
// expected: '+61438331999' (as per Google)
// actual: '+6161438331999'
```

See Google's [result](https://libphonenumber.appspot.com/phonenumberparser?number=61438331999&country=AU)

**Comment by catamphetamine** [comment]
This topic has been discussed before: https://github.com/catamphetamine/libphonenumber-js/issues/316
What is a scenario in which such error correction would be required?

**Comment by imranariffin** [comment]
Thanks for the swift response @catamphetamine. No specific scenario actually but after going ovrr the discussion you mentioned it seems kike this

**Comment by catamphetamine** [comment]
@imranariffin Actually, released `libphonenumber-js@1.7.41` with the leading `+` autocorrection feature.

**Comment by imranariffin** [comment]
> @imranariffin Actually, released `libphonenumber-js@1.7.41` with the leading `+` autocorrection feature.

I see, thanks for the update. I was using `1.7.31` so that makes sense. I will try upgrading and testing it again.
